core/src/main/scala/caliban/wrappers/ApolloPersistedQueries.scala

```diff
@@ -42,13 +57,12 @@ object ApolloPersistedQueries {
           readHash(request) match {
             case Some(hash) =>
               ZIO
-                .accessM[ApolloPersistence](_.get.get(hash))
+                .serviceWith[Service](_.get(hash))
                 .flatMap {
-                  case Some(query) => UIO(request.copy(query = Some(query)))
-                  case None        =>
+                  case Some(doc) => docVar.set(Some(Right(doc))) as request
```

Comment:
Do we prefer `as x` over `.as(x)`?

Reply:
I'm not sure, I don't think we have a style guide for it, but I'm fine with either I know the infix can get a little out of hand sometimes.

Reply:
I tend to prefer `.as(x)` and `.` in general except when there's a symbol alias.

Reply:
@ghostdogpr that's usually what I do as well but I don't really have any strong opinions around it so I'm fine with either